### PR TITLE
Add `-c` alias for `--config`

### LINKED ISSUE #6690: Add `-c` alias for `--config`
requested by @ybiquitous in https://github.com/stylelint/stylelint/pull/6617#discussion_r1121431016

see https://github.com/stylelint/stylelint/blob/4835d16d08fd91ff9e0ab47c1f7adbcf73fb0342/lib/cli.js#L261

### PR DESCRIPTION
**Pull request by sidverma32**
> Which issue, if any, is this issue related to?

Closes #6690 

> Is there anything in the PR that needs further explanation?

Refer to this discussion:
https://github.com/stylelint/stylelint/pull/6617#discussion_r1121431016)

### CODE REVIEW

**Review by ybiquitous — CHANGES_REQUESTED** [review]
[suggest] Can you add a new changelog entry with this PR title? The [changeset-bot instruction](https://github.com/stylelint/stylelint/pull/6720#issuecomment-1476036536) should be helpful.

Like this:

```yaml
---
"stylelint": minor
---

Added: `-c` alias for `--config`
```

File: lib/__tests__/__snapshots__/cli.test.js.snap [hunk 1]
```diff
@@ -14,7 +14,7 @@ exports[`CLI --help 1`] = `
 
   Options:
 
-    --config
+    --config , -c
```
Comment (Mouvedia) [inline lib/__tests__/__snapshots__/cli.test.js.snap:17]:
```suggestion
    --config, -c
```

**Review by ybiquitous — APPROVED** [review]
Thanks, LGTM 👍🏼

**Comment by sidverma32** [comment]
> Thanks, LGTM 👍🏼

Shall I merge?

**Pull request merged**